Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml

```diff
@@ -20,6 +20,17 @@
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpskirt/captain.rsi
 
+- type: entity
+  parent: [ClothingUniformBase, BaseCommandContraband]
+  id: ClothingUniformJumpskirtCaptainTurtleneck
+  name: captain's turtleneck
+  description: A regal blue turtleneck finely made with mothroach silk. Fit for a captain.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpskirt/capturtleskirt.rsi
+
```

Comment:
This entity should be moved to `Resources/Prototypes/_Harmony/Entities/Clothing/Uniforms/jumpskirts.yml`